syntax = "proto3";

package engine;

message TilesheetParams {
    string atlas_filename  = 1;
    int32 tile_size        = 2;
    int32 tiles_width      = 3;
    int32 tiles_height     = 4;
    int32 atlas_gutter     = 5;
    string fragment_shader = 6;
    string vertex_shader   = 7;
}

message EngineParams {
    int32 window_width  = 1;
    int32 window_height = 2;
    int32 render_scale  = 3;
}
